test/index.js

```diff
@@ -1,74 +1,70 @@
-const sinon = require('sinon');
-const assert = require('assert');
-const printTree = require('../index');
+// @flow
 
-function validateTree(consoleFunc, tree) {
-  tree.forEach((branch, index) => {
+import test from 'ava';
+import sinon from 'sinon';
+import printTree from '../index';
+
+function validateTree(t, tree, output) {
```

Comment:
This is using the concept of [Test macros](https://github.com/avajs/ava#test-macros) in AVA